sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -520,16 +488,15 @@ client.sendTypingNotification(chatThreadId: threadId) { result, _ in
 
 #### Send read receipt
 
-Use the `send` method to post a read receipt event to a thread, on behalf of a user.
+Use the `send` method of `ChatThreadClient` to post a read receipt event to a thread, on behalf of a user.
 
 - `SendReadReceiptRequest` is the model to be passed to this method.
 - `chatMessageId` is the unique ID of the message.
-- `chatThreadId` is the unique ID of the thread.
 
 ```swift
 let readReceipt = SendReadReceiptRequest(chatMessageId: messageId)
 
-client.send(chatReadReceipt: readReceipt, chatThreadId: threadId) { result, _ in
+chatThreadClient.send(chatReadReceipt: readReceipt) { result, _ in
```

Comment:
Maybe just `send(readReceipt:)`

Reply:
This is another case where the model only has one property. So what about something like `sendReadReceipt(forMessageWithId:)` (or `sendReadReceipt(forMessage:)`) ?